.github/workflows/CI.yaml

```diff
@@ -219,6 +223,8 @@ jobs:
         config:
           - name: "keptn-lifecycle-operator"
             folder: "operator/"
+          - name: "klt-metrics-operator"
```

Comment:
I would suggest to drop the `klt-` prefix for the `metrics-operator` in general, as the containing directory and the go module do not have that either

Reply:
fixed